test/fixtures/config/config-with-name-and-tag-template.yml

```diff
@@ -0,0 +1,3 @@
+template: Placeholder with example. Automatically calculated values based on previous releases are next major=$NEXT_MAJOR_VERSION, minor=$NEXT_MINOR_VERSION, patch=$NEXT_PATCH_VERSION.
+name-template: 'v1.2.3 (Code name: Hello World)'
+tag-template: v3.4.5
```

Comment:
? Could we use some $RESOLVED_VERSION so at least the test looks sane 😓 I cannot handle that your getting one version in name and a different in tag.